packages/workbox-webpack-plugin/src/lib/utils/resolve-webpack-url.js

```diff
@@ -0,0 +1,29 @@
+/*
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+/**
+ * Resolves a url in the way that webpack would (with string concatenation)
+ *
+ * Use publicPath + filePath instead of url.resolve(publicPath, filePath) see:
+ * https://webpack.js.org/configuration/output/#output-publicpath
+ *
+ * @function resolveWebpackUrl
+ * @param {Array<string>} paths File paths to join
+ * @return {string} Joined file path
+ *
+ * @memberof module:workbox-webpack-plugin
+ */
+module.exports = (...paths) => paths.join('');
```

Comment:
This would get you in trouble, because all webpack plugins I've used so far are capable of handling public paths that do not end with a `/`, including webpack itself.

Reply:
There was discussion about this. This is why I isolated the path logic too. Do you think we should do a `url.resolve` rather that `String.join`?

Reply:
Maybe keep it as it is. It is quite obvious what's wrong when you see a slash missing between publicPath and filename.

Reply:
Yes, I'm the one who keeps making an issue out of this 😄 

It seems like we should be following the conventions from the Webpack docs, which says that `publicPath` is a string prefix, but I can stop making a big deal about it if folks think that in practice, it's always joined with a path separator.

Reply:
I'll just make note to make sure the documentation is clear on the path logic incase there is any trouble. We can revisit using `url.resolve()` if this is an issue.